graylog2_handler/__init__.py

```diff
@@ -0,0 +1,32 @@
+import openerp.tools as tools
+import logging
+
+_logger = logging.getLogger(__name__)
+
+# Check if this module is enabled in config
+if tools.config.get('gelf_enabled', False):
+    try:
+        import graypy
+    except ImportError:
+        _logger.error("Package 'graypy' not installed")
+    else:
```

Comment:
if next code is moved out from ```else``` block, and ```graypy``` is not installed, then error will be raised on import of this module. But if ```else``` block is present, just an error message will be written in log. no errors will be raised.